static/js/user_status_ui.js

```diff
@@ -82,6 +82,12 @@ exports.initialize = function () {
         exports.submit_new_status();
     });
 
+    $('body').on('keypress', '.user_status_overlay .user_status', function (e) {
+        if (e.key === 'Enter') {
+            exports.submit_new_status();
+        }
```

Comment:
Should this have a `preventDefault` / `stopPropagation` for better robustness?

Reply:
Also is this cleaner here or in `hotkey.js`?

Reply:
>Should this have a preventDefault / stopPropagation for better robustness?
Yes - added it in the latest force push.

>Also is this cleaner here or in hotkey.js?
I'm not sure, but `alert_word_ui.js:92` adds the event listener in the same file, so atleast this is consistent.

```js
    $('#alert_words_list').on('keypress', '#create_alert_word_name', function (event) {
        const key = event.which;
        // Handle enter (13) as "add".
        if (key === 13) {
            event.preventDefault();

            const word = $(event.target).val();
            add_alert_word(word);
        }
    });
```

Reply:
( On looking at that file, event.which is Deprecated, we should change that to event.key )

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which

Reply:
Yeah, we should do that migration sometime.